Resources/Prototypes/Entities/Mobs/NPCs/regalrat.yml

```diff
@@ -131,6 +131,8 @@
       gender: male
   - type: Sharp
   - type: PotentialPsionic # Nyano
+  - type: NightVision #Delta-v - Added nightvision to rat king
+    drawOverlay: false
```

Comment:
Suggestion: Make the color not green. The color in the suggestion is a grey color.

```suggestion
  - type: NightVision #Delta-v - Added nightvision
    drawOverlay: false
    color: "#808080"
```